dockerfiles/bash/Dockerfile

```diff
@@ -3,24 +3,87 @@ WORKDIR /go/src/github.com/sourcegraph/lsp-adapter
 COPY . .
 RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-adapter
 
-# 👀 Add steps here to build the language server itself 👀
-# CMD ["echo", "🚨 This statement should be removed once you have added the logic to start up the language server! 🚨 Exiting..."]
+FROM mongo:3.7.9
 
-FROM node:9
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y curl=7.47.0-1ubuntu2.8 \
+  && rm -rf /var/lib/apt/lists/*
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y \
+    nodejs=8.11.2-1nodesource1 \
```

Comment:
Is there a reason that we're using node version `8`?

Reply:
Nope, it just seems to work

Reply:
If it works lets use 10. They are both LTS